apache/trafficcontrol
# Tp interface servers

## What does this PR (Pull Request) do?
- [x] This PR fixes #4623 and fixes #4792 

Updates Traffic Portal tables and forms to account for servers having multiple interfaces, including:

- any table that shows servers obtained from `/servers` such as
    - Servers
    - Cache Groups -> View Servers
    - Profiles -> View Servers
    - Delivery Services -> View Servers
- The server create/edit form

## Which Traffic Control components are affected by this PR?
- Traffic Portal

## What is the best way to verify this PR?
- Build Traffic Portal, verify that all the mentioned tables and forms work properly.
- Run the TP integration tests.
- You could also run the UI tests, but I think they're broken on master atm _[citation needed]_, so I'll need to look into that.
## If this is a bug fix, what versions of Traffic Control are affected?

## The following criteria are ALL met by this PR
